src/Microsoft.Crank.RegressionBot/Program.cs

```diff
@@ -271,7 +271,7 @@ private static async Task CreateRegressionIssue(IEnumerable<Regression> regressi
 
             var body = await CreateIssueBody(regressions, template);            
             
-            var title = "Performance regression: " + String.Join(", ", regressions.Select(x => x.CurrentResult.Scenario).Take(5));
+            var title = "Performance variation: " + String.Join(", ", regressions.Select(x => x.CurrentResult.Scenario).Take(5));
```

Comment:
change?

Reply:
I like it, also deviation 

Other options: difference · disparity · imbalance · alteration · modification · variability · variance · divergence · fluctuation · modification

Reply:
difference